testing/ase/test_sw_c_ase.cpp

```diff
@@ -542,3 +490,34 @@ TEST(LibopaecAseOps, ase_app_01) {
 	remove_existing_lock_file("app_test2.txt");
 }
 
+/**
+* @test       ase_app_02
+*
+* @brief      When the parameters are nullptr,
+*             is_directory function returns 0;
+*
+*/
+TEST(sim_sw_ase, ase_app_02) {
+	EXPECT_EQ(0, is_directory(nullptr));
+}
+
+/**
```

Comment:
Move lines 504-510 to line 515, or else doxygen will not render correctly for this test.